.circleci/config.yml

```diff
@@ -86,3 +86,11 @@ jobs:
             chmod +x ./cc-test-reporter
             ./cc-test-reporter before-build
             ./cc-test-reporter after-build
+
+      - run:
+          name: Check for missing annotations
+          command: RAILS_ENV=test bundle exec annotate
+      
+      # Check if running annotate changed any file
+      - check_untracked_files:
```

Comment:
I'm not sure how this yml should be structured because I don't remember having to configure it myself.

But I see that you have 2 `check_untracked_files` items at the same level, and both have different `path`. Is that Ok? Does it run from the top down?

Also another thing is that you put the path to check for annotations to be `.`, and maybe you can set it to `app/models`?

Reply:
Yeah, my mistake I didn't clarify, the cost of running the `check_untracked_files` check is almost 0 seconds. As the build follows a top-down strategy, I could have placed it only at the end of the build and check if any files changed, but this way (running it several times (3)) it is more clear what the dev is missing (schema/annotations), because it will say specifically what it was checking.

WDYT?

Annotations are in the models, also specs I think, and in factories. Maybe I am missing some more